helm_deploy/templates/deployment-worker.yaml

```diff
@@ -10,15 +10,15 @@ spec:
   {{- end }}
   selector:
     matchLabels:
-      {{- include "laa-crime-application-store.selectorLabels" . | nindent 6 }}
+      app:  {{ include "laa-crime-application-store.fullname" . }}-worker
```

Comment:
should we remove the `selectorLabels` helper entirely and update `deployment.yaml` to use the same, minus the `-worker`

```yml
  selector:
    matchLabels:
      app: {{ include "laa-crime-application-store.fullname" . }}
  template:
    metadata:
      labels:
        app: {{ include "laa-crime-application-store.fullname" . }}
```

But, this would provide different output of which i am not sure of the impact, namely instead of `app`  the selectorLabels helper interpolates `app.kubernetes.io/name` and adds `app.kubernetes.io/instance` too.

```
app.kubernetes.io/name: {{ include "laa-crime-application-store.name" . }}
app.kubernetes.io/instance: {{ .Release.Name }}